proseco/acq.py

```diff
@@ -848,6 +935,46 @@ def calc_p_brightest(self, acq, box_size, man_err=0, bgd=0):
 
         return prob
 
+    def get_overlap_penalties(self):
+        """
+        Get the penalties for overlapping boxes.
+
+        :returns: list of penalties (float)
+        """
+        n_acq = len(self)
+        penalties = np.ones(n_acq)
```

Comment:
I think the default "penalty" makes more sense as 0.0 , with the penalty then applied as `* 1 - penalty` but that's mostly semantic.

Reply:
Thanks!